docs/modules/whats-new/whats-new.adoc

```diff
@@ -0,0 +1,10 @@
+What's new in Platform 5.4
+
+First of all, starting with this release of Hazelcast Platform, the minimum supported Java version has become 17.
+
+You can now deploy, undeploy or redeploy your custom classes by using the user code name spacing feature. The feature provides a container for Java classpath resources, such as your classes, and accompanying artifacts like property files - enabling namespace isolation to ensure that access to resources in different namespaces can be managed through configuration.
```

Comment:
```suggestion
You can now deploy, or redeploy your custom classes by using the user code name spacing feature. The feature provides a container for Java classpath resources, such as your classes, and accompanying artifacts like property files - enabling namespace isolation to ensure that access to resources in different namespaces can be managed through configuration.
```

Reply:
There was a late change that meant you cannot undeploy at runtime, the agreement was to focus on deploying and redeploying